sdk/tables/data-tables/src/TableClient.ts

```diff
@@ -162,7 +158,7 @@ export class TableClient {
     if (isInternalClientOptions(clientOptions)) {
       // The client is meant to be an intercept client (for Transaction), so we need to create only the intercepting
       // pipelines.
-      internalPipelineOptions.pipeline = clientOptions.innerTransactionRequest.createPipeline();
+      internalPipelineOptions.pipeline = createEmptyPipeline();
```

Comment:
The comment above now seems wrong since we no longer create an intercepting pipeline, can you update it to explain what the new strategy is?

Reply:
Oh yeah, good catch! I'll remove the condition altogether since we no longer need it

Reply:
Okay, @richardpark-msft gave me the idea of having these reset-able properties as an object and the initialize function to return the initialized object. I think it gets us a better grouping on what properties may be reset and also allows for sharing code. Thanks, guys for the help improving this 😄